### PR TITLE
Beta3

### PR DESCRIPTION
**Pull request by asjafjell**
Introduces the Beta3 XSD and all the happy changes that comes with it.

The important changes are:
- Sender can be defined in `ClientConfiguration` and in each `DirectJob` and `PortalJob`. 
- Availability is defined by an `Activation` and `AvailableSeconds`, not `Activation` and `Expiration` as earlier.
- Polling for changes for `DirectJob` now requires a token which will be a query-parameter in the `ExitUrl` the signer was redirected to.

### CODE REVIEW

File: Digipost.Signature.Api.Client.Core/Internal/CreateAction.cs [hunk 1]
```diff
@@ -35,6 +38,8 @@ internal override HttpContent Content()
 
         private StringContent BodyContent()
         {
+            Log.Debug($"Outgoing body: {SerializedBody}");
```
Comment (SindreNordbo) [inline Digipost.Signature.Api.Client.Core/Internal/CreateAction.cs]: This should probably be replaced by configurable request logging :)

File: Digipost.Signature.Api.Client.Direct.Tests/Smoke/DirectClientSmokeTests.cs [hunk 2]
```diff
@@ -73,34 +80,46 @@ public static void CreateAndGetStatus(TestContext context)
...
 
-                //Act
-                var result = await directClient.Create(directJob);
+                ////Act
+                //var result = await directClient.Create(directJob);
 
-                //Assert
-                Assert.IsNotNull(result.JobId);
+                ////Assert
+                //Assert.IsNotNull(result.JobId);
```
Comment (SindreNordbo) [inline Digipost.Signature.Api.Client.Direct.Tests/Smoke/DirectClientSmokeTests.cs]: Uncomment :dancer:

File: Digipost.Signature.Api.Client.Direct.Tests/StatusReferenceTests.cs [hunk 3]
```diff
@@ -13,13 +13,14 @@ public class ConstructorMethod : StatusReferenceTests
...
+                var urlWithoutToken = new Uri("http://organizationdomain.com/completionUrl/");
+                var statusQueryToken = "ALongToken";
 
                 //Act
-                var directJobReference = new StatusReference(jobReference);
+                var statusReference = new StatusReference(urlWithoutToken, statusQueryToken);
 
                 //Assert
-                Assert.AreEqual(jobReference, directJobReference.Url);
+                //Assert.AreEqual();
```
Comment (SindreNordbo) [inline Digipost.Signature.Api.Client.Direct.Tests/StatusReferenceTests.cs]: Same as above, an assert here seems like a good idea?

**Comment by SindreNordbo** [comment]
Looks good. Review the comments and merge at will :)

**Comment by asjafjell** [comment]
Magnificent!

**Pull request merged**